app/helpers/routes_into_teaching_helper.rb

```diff
@@ -5,16 +5,16 @@ def answers
 
   def undergraduate_degree_summary
     {
-      "Yes" => "have a degree",
-      "No" => "do not have a degree",
-      "Not yet" => "are studying for a degree",
+      "Yes" => "have a bachelor's degree",
+      "No" => "do not have a bachelor's degree",
+      "Not yet" => "are studying for a bachelor's degree",
     }[answers["undergraduate_degree"]]
   end
 
   def unqualified_teacher_summary
     {
       "Yes" => "have previously worked in a school",
-      "No" => "haven't previously worked in a school",
+      "No" => "have not previously worked in a school",
```

Comment:
Not sure I understand this change - this is not something that appears on the front end?

Reply:
Ignore this - this is on the completion page!